### PR TITLE
Automate rn / flutter release pipelines

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -89,3 +89,56 @@ jobs:
...
+      - name: Authenticate
+        run: |
+          mkdir -p $PUB_CACHE
+          cd $PUB_CACHE
+          echo "$PUBDEV_CREDENTIALSJSON" > credentials.json
+        env:
+          PUBDEV_CREDENTIALSJSON: ${{ secrets.PUBDEV_CREDENTIALSJSON }}
+
+      - name: Publish to pub.dev
+        run: dart pub publish --force # TODO: automatically update CHANGELOG to run this without --force
```
Comment (bangtoven) [inline .github/workflows/deploy.yml:140]: image

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -89,3 +89,56 @@ jobs:
...
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Publish to npm
+        run: |
+          npm install
+          npm publish
```
Comment (bangtoven) [inline .github/workflows/deploy.yml:113]: image

**Review by AMITGOELNY — DISMISSED** [review]

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -89,3 +88,56 @@ jobs:
...
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Publish to npm
+        run: |
+          npm install
```
Comment (vishnumad) [inline .github/workflows/publish.yml:111]: We should run the `prepare` script before `npm publish`

```
npm install
npm run prepare
npm publish
```

Comment (bangtoven) [reply .github/workflows/publish.yml:111]: 👍

**Review by vishnumad — APPROVED** [review]

**Pull request merged**